lib/components/src/blocks/ArgsTable/SectionRow.stories.tsx

```diff
@@ -5,7 +5,7 @@ import { ResetWrapper } from '../../typography/DocumentFormatting';
 
 export default {
   component: SectionRow,
-  title: 'Docs/SectionRow',
+  title: 'Docs/SectionRow/Args',
```

Comment:
This was warning about duplicate stories being added.

Reply:
ArgsSectionRow?